tests/responses/ok-test.js

```diff
@@ -113,7 +115,58 @@ test('Returns correct attributes', function (t) {
   });
 });
 
-test('Returns correct nested resources', function (t) {
+test('Fetching records collection: many-to relationship', function (t) {
+  t.plan(6);
+
+  sails.request({
+    url   : '/author',
+    method: 'GET',
+  }, function (err, res, body) {
+    if (err) {
+      t.fail(err);
+    }
+    try {
+      t.equal(res.statusCode, 200, 'HTTP status code is 200');
+      t.equal(res.headers['Content-Type'], 'application/vnd.api+json', 'Sends jsonapi mime type');
+      t.ok(validateJsonApi(body), 'Body is a valid JSON API');
+      t.ok(body.data[0].relationships, 'Resource object has relationships object');
+      t.ok(_.isArray(body.data[0].relationships.books.data), 'Relationship value is an array for many-to relationship');
+      t.deepEqual(body.data[0].relationships.books.data[0], { type: 'books', id: '1' }, 'Relationship data is an resource linkage as default');
+    } catch (err) {
```

Comment:
@IanVS will investigate that one and if possible remove try/catch blocks from all tests in another PR.